pkgs/top-level/cuda-packages.nix

```diff
@@ -0,0 +1,137 @@
+{ lib
+, pkgs
+, stdenv
+, cudaVersion
+}:
+
+with lib;
+
+(makeScope pkgs.newScope (self: let
+
+  callPackage = self.callPackage;
+
+  buildCudaRedistPackage = callPackage ../development/compilers/cudatoolkit/redist/build-cuda-redist-package.nix { };
+
+  # Manifest files for redist cudatoolkit
+  manifests = {
+    "11.4" = ../development/compilers/cudatoolkit/redist/manifests/redistrib_11.4.4.json;
+    "11.5" = ../development/compilers/cudatoolkit/redist/manifests/redistrib_11.5.2.json;
+    "11.6" = ../development/compilers/cudatoolkit/redist/manifests/redistrib_11.6.2.json;
+  };
+
+  buildCudaPackage = pkgs.callPackage ../development/compilers/cudatoolkit/common.nix;
+
+  # Classic cudatoolkit packages that contain everything that is in redist.
+  cudatoolkitVersions = {
```

Comment:
prefer to put this in a toml file so it can be easily updated both by human and script